docs/pages/templates/typescript/_meta.js

```diff
@@ -1,5 +1,4 @@
 export default {
-  "getting-started": "Getting Started",
```

Comment:
reminder that we should always add a redirect if we break a link

Reply:
Thank you for reminding me. Done.